handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -601,6 +605,11 @@ public final SSLEngineResult wrap(
                 }
             }
 
+            if (dst.remaining() < MAX_ENCRYPTED_PACKET_LENGTH) {
```

Comment:
Is it correct to use a constant here or should we get this value from the session? Or does the session always return this constant?